src/clientiface.cpp

```diff
@@ -302,6 +305,9 @@ void RemoteClient::GetNextBlocks (
 			if (m_blocks_sent.find(p) != m_blocks_sent.end())
 				continue;
 
+			/*
+				Or blocks we already have occlusion culled
```

Comment:
actually there's a `block->resetUsageTimer();` further down which is probably not safe to skip
I suggest moving this check closer to the `isBlockOccluded` call

Reply:
Thanks @sfan5 .

I can see this either way...
We also do not reset the usage time if the block is known to the client (in m_blocks_sent).

In any, case, moved it below the call to resetUsageTimer